Surf 1149 when to engage helix

araiford opened the pull request:
Added Engaging Helix article to the website.

I figure we'll soon add a section to this page describing How to engage Helix, so I left the page title a bit more high level

The page lives under Resources, which required moving the current resources content to a Libraries & Repo's page.  Let me know you thoughts on this.

araiford commented:
### screenshot

![image](https://user-images.githubusercontent.com/933210/42585675-ec049a8c-84fb-11e8-8f42-fda8d701ed4d.png)

nimbinatus commented on _docs/resources/libraries-and-repos.md:
```diff
@@ -0,0 +1,123 @@
+---
+title: Libraries & Repo's
```
s/`Repo's`/`Repos`

araiford replied:
👍

nimbinatus commented on templates/_includes/nav.html:
```diff
@@ -15,9 +15,9 @@
+      {%- if currentNav == "style" or currentNav == "resources" -%}
         {%- for article in site.docs -%}
-          {%- if article.category == "Style" -%}
+          {%- if article.category | slugify == currentNav" -%}
```
Trailing extra `"` at the end can be removed.

araiford replied:
👍

nimbinatus commented on _docs/resources/sketch-library.md:
```diff
@@ -2,8 +2,9 @@
-resource: true
+resource: false
 category: Resources
+parent: Libraries & Repo's
```
I take it we'll use the `parent` call at some point for the Resources section? Or is this just to add it in case?

araiford replied:
Setting the parent to Libraries & Repos enables us to highlight that nav item, while still keeping the child page (sketch-library.md) invisible from the left nav.  This is how the Text Conventions articles are handled as well.

When we move to one site, we should consider whether we might need a tertiary level of navigation for things like this.

nimbinatus commented on _docs/resources/engaging-helix.md:
```diff
@@ -0,0 +1,98 @@
+category: Resources
+last-modified: 2018-07-11
+usage: >
+   The long and short of it is that you should reach out the the Helix team anytime the team might be helpful to you, but it’s not always easy to know what that is. Use the following scenarios as a guide for when you should contact the Helix team
```
Needs a period at the very end.

araiford replied:
👍

chri2547 commented on _docs/resources/engaging-helix.md:
```diff
@@ -0,0 +1,98 @@
+---
+title: Engaging Helix
```
To me, this title is ambiguous - is it about engaging with helix assets (design language and code), or engaging with helix team members? Seems like the latter. Maybe consider titling this "Engaging the Helix Team" or something along those lines?

araiford replied:
good point!

palindromebart replied:
+1

chri2547 commented on _docs/resources/engaging-helix.md:
```diff
@@ -0,0 +1,98 @@
+category: Resources
+last-modified: 2018-07-11
+usage: >
+   The long and short of it is that you should reach out the the Helix team anytime the team might be helpful to you, but it’s not always easy to know what that is. Use the following scenarios as a guide for when you should contact the Helix team.
```
I'm not sure what to do with this content, tbh. It's located in an area of the page that most folks probably overlook due to banner blindness. Even so, I would go one of two directions with this content. I would:
1) make it super plain and say something like "Use the following scenarios as a guide..." - short and sweet.
2) expand this text to include more information about Helix and the Helix team. I feel like this page is missing that context. Maybe the 'About' information could be placed further down the page, before the cartoons? There may be new team getting on board, or new hires, that could benefit from that kind of elevator pitch.

palindromebart replied:
I agree with Chris' assessment. For the moment, I think we should go with option 1.

chri2547 commented on _docs/resources/engaging-helix.md:
```diff
@@ -0,0 +1,98 @@
+usage: >
+   The long and short of it is that you should reach out the the Helix team anytime the team might be helpful to you, but it’s not always easy to know what that is. Use the following scenarios as a guide for when you should contact the Helix team.
+---
+
```
Maybe this is where we could include a short and sweet About section? Maybe at the end of the About section you can list the ways in which I can engage with the Helix team? Like, maybe I already know I have a legit reason to engage, so how do I do that? If someone still has questions about when to engage, they can read the awesome cartoons.

palindromebart commented on _docs/resources/engaging-helix.md:
```diff
@@ -0,0 +1,98 @@
+
+{% column left:"hxCol hxSpan-4" %}
+
+![When I want to know if I’m using Helix correctly or not]({{site.url}}/assets/images/resources/engaging-helix/feedback-and-review-using-helix-correctly.png){: width="100%"}
```
Agree with Laura's comments about the supporting text occupying a weird place in the info hierarchy. Maybe we can add the text into the image? Like 
<img width="525" alt="screen shot 2018-07-13 at 3 40 20 pm" src="https://user-images.githubusercontent.com/5124722/42712922-19c27c06-86b3-11e8-9fc5-9c095ae63927.png">

araiford commented:
@nimbinatus @chri2547 @palindromebart 

Updates made to the When to Engage Helix article.

After reading feedback on the PR, I chose to move this article out into a page which is available from left nav via "CONTACT US".

I added a section for "How to contact us".  The "Request a new design pattern" link on the far right temporarily points at the GitHub issue template, which I've revised in this PR to include the basic questions from our intake process.  Ideally, we can point this link at a Jira intake form until we move to one-site.

I've also added a section for "About the team", with some nifty avataaaaars for each of us.  I'm totally up for customizing these avatars so they look more like us if we like this approach.

Anyway, take a gander and let me know your thoughts.

palindromebart commented:
OMG!! I love the avataaars!!

araiford commented:
@chri2547 @palindromebart -- updated contact us article per feedback.  Also, this gets the avatars off the CDN, so if anyone feels like changing their avatar we don't have to deploy code.

nimbinatus commented:
@araiford I'll have some code updates for you (probably tomorrow). Would you like me to leave comments here, or would you rather have a PR against your branch? Either one works for me.

araiford commented:
@nimbinatus -- go ahead and leave the comments here.

nimbinatus commented on _docs/contact-us/index.md:
```diff
@@ -0,0 +1,126 @@
+{% include toc.html %}
+</section>
+
+
```
Any reason for all of the extra space here?

nimbinatus commented on _docs/contact-us/index.md:
```diff
@@ -0,0 +1,126 @@
+---
+
+<section class="static-section" markdown="1" style="margin-bottom:4rem;">
+{% include toc.html %}
```
Global: Need dashes in the nunjucks calls to avoid space in the rendered HTML:
e.g. s/`{% include toc.html %}`/`{%- include toc.html -%}`

araiford replied:
ahhhh -- I didn't know what those dashes were for!  Thanks for telling me.

nimbinatus commented on _docs/contact-us/index.md:
```diff
@@ -0,0 +1,126 @@
+
+{% for illoSection in site.data.contact-helix-illos %}
+{% assign sectionTitle = illoSection[0] %}
+{% assign illustrations = illoSection[1]%}
```
s/`{% assign illustrations = illoSection[1]%}`/`{%- assign illustrations = illoSection[1] -%}`

nimbinatus commented on _docs/contact-us/index.md:
```diff
@@ -0,0 +1,126 @@
+{% column left:"hxCol hxSpan-3" %}
+
+#### <a href="https://github.com/rackerlabs/design-system/issues/new?title=Issue%20Regarding%20-%20[Component Name Here]" target="_blank">Report problems with a design <hx-icon type="external-link"></hx-icon></a>
+Create an issue on the Helix website repo, and outline the problem.  We'll triage it and let you know next steps.
```
new line between 77 and 78

nimbinatus commented on _docs/contact-us/index.md:
```diff
@@ -0,0 +1,126 @@
+{% column left:"hxCol hxSpan-3" %}
+
+#### <a href="https://github.com/rackerlabs/design-system/issues/new?title=New%20Design%20Pattern%20Request%20-%20[Request Name Here]" target="_blank">Request a new design <hx-icon type="external-link"></hx-icon></a>
+Submit a request through GitHub and we'll get back to you asap with next steps and objectives.
```
new line between 84 and 85

nimbinatus replied:
s/`asap`/`ASAP`

nimbinatus commented on _docs/contact-us/index.md:
```diff
@@ -0,0 +1,126 @@
+<div class="avatar-list">
+{% for member in sub %}
+<div class="avatar-item">
+  <img class="avatar-item-image" src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/assets/images/contact-us/{{ member[1] | downcase | replace: ' ','-' }}.png">
```
This link will point to a CDN. If you want to remove the CDN as noted in comments, use this:
```
  <img class="avatar-item-image" src="{{site.url}}/assets/images/contact-us/{{ member[1] | downcase | replace: ' ','-' }}.png">
```

nimbinatus commented on assets/css/_sass/website/_shame.scss:
```diff
@@ -68,3 +72,69 @@ p {
 }
+
+
+.avatar-list {
```
I think we can move these items specifically for the contact us page to their own file rather than living in the `_shame.scss` file. It's not really a shame category to me. Up to you, though.

araiford commented:
@nimbinatus @palindromebart @chri2547 -- updates made, looking for the 👍
